CHANGELOG.md

```diff
@@ -4,10 +4,12 @@ Changelog for NeoFS Node
 ## [Unreleased]
 
 ### Added
+- SN eACL processing of NULL and numeric operators (#2742)
```

Comment:
the same commit as in https://github.com/nspcc-dev/neofs-node/pull/2733 but with a different code?

Reply:
> the same commit

what do u mean?